content/2016.09.fullstack-fest-en.rst

```diff
@@ -0,0 +1,195 @@
+
+- Evolution of Web apps which now have views, business logic and persistence, everything has been about improving the developer experience — `Luca Marchesini <https://www.youtube.com/watch?v=wtURpqTgtUs&index=23&list=PLe9psSNJBf76DOOKMkDpyo_A5PfZk7JWc>`_, Kuzzle
+
+- Om Next leverages Clojure to bring conventions for data immutability, data specifications and data routing — `António Nuno Monteiro <https://www.youtube.com/watch?v=Zb18iPjDgwM&index=24&list=PLe9psSNJBf76DOOKMkDpyo_A5PfZk7JWc>`_, Technische Universität Dresden
+
+- The OpenBCI project allows everyone to study brainwaves using a homemade helmet —  `Alex Castillo <https://www.youtube.com/watch?v=CSfUr3m0-w8&index=25&list=PLe9psSNJBf76DOOKMkDpyo_A5PfZk7JWc>`_, Netflix
+
+- Flow based programming via CycleJS allows you to design and debug your apps via a high level of abstraction — `André Staltz <https://www.youtube.com/watch?v=R-GzJgEccEQ&list=PLe9psSNJBf76DOOKMkDpyo_A5PfZk7JWc&index=27>`_, Futurice
+
+- CSP channels in Javascript are the best tooling when dealing with stream of data — `Vincenzo Chianese <https://www.youtube.com/watch?v=r7yWWxdP_nc&index=28&list=PLe9psSNJBf76DOOKMkDpyo_A5PfZk7JWc>`_, Apiary
+
+- Service workers allow us to forget about page loader, especially when submitting data — `Andrew Dunkman <https://www.youtube.com/watch?v=xs_QRqGZ8xQ&index=29&list=PLe9psSNJBf76DOOKMkDpyo_A5PfZk7JWc>`_, Harvest
+
+- Build your apps as if there were be accessed by Martian via a high-latency connection ­— `Slobodan Stojanovic <https://www.youtube.com/watch?v=7rlEidtXlZg&index=30&list=PLe9psSNJBf76DOOKMkDpyo_A5PfZk7JWc>`_, Cloud Horizon
+
+- WebAssembly allows native apps to be run by the browser VM in the same stack context as JavaScript — `Ben Smith <https://www.youtube.com/watch?v=vmzz17JGPHI&index=31&list=PLe9psSNJBf76DOOKMkDpyo_A5PfZk7JWc>`_, Google
+
+- Render a 2D UI with React within a 3D WebGL scene — `Massimiliano Mantione <https://www.youtube.com/watch?v=DfPPlakRvow&index=32&list=PLe9psSNJBf76DOOKMkDpyo_A5PfZk7JWc>`_, HyperFair
+
+- Virtual reality in the browser is here! — `Liv Erickson <https://www.youtube.com/watch?v=Ciqucr_Ww9s&index=33&list=PLe9psSNJBf76DOOKMkDpyo_A5PfZk7JWc>`, Microsoft
+
+- The physical Web on top of the open Web and will change our lives — `Scott Jenson <https://www.youtube.com/watch?v=gV72mCdomo4&index=34&list=PLe9psSNJBf76DOOKMkDpyo_A5PfZk7JWc>`_, Google
+
+
+Some thoughts
+=============
+
+I felt really concerned by the challenges Joe and Juan raised! The Web of today has too many points of failure, Web engineers are responsible of making it better. For example, 500 attendees cannot collaborate on a document without going through servers outside the room! My grand-grand-grand children might not be able to find a picture of me on the Web! `But read this page <https://ipfs.io/#why>`_ and you'll realize this is not *only* about the future!
```

Comment:
I had to use my browser's "search" functionality to find who you meant by Joe and Juan. (I know Joe Armstrong, but the name "Joe" is so common that I didn't thin immediately of him.) I'm not sure what you can do here without making this sentence too heavy.

Something about the first sentence here bothers me, maybe because "concerned" here makes me think of being involved with something, or maybe because "I felt ... by" seems odd. "The challenges Joe and Juan raised made me feel really concerned" sounds a little better. How about "The challenges Joe and Juan raised are very concerning"?

"responsible for", not "responsible of".

"great-great-grandchildren".